packages/cli/src/utils.ts

```diff
@@ -298,5 +298,8 @@ export function getColorizedString(message: Array<{ content: string; color: Colo
       }
     })
     .join("");
-  return colorizedMessage + "\u00A0";
+  // Pad the last line to avoid tailing color
+  const padWidth = (s: string, stream = process.stdout) =>
+    s + " ".repeat(stream.columns - (s.length % stream.columns));
```

Comment:
If this msg is not beginning at the left of the command line, the padding width will be wrong.

Reply:
Do you mean like `[TeamsFx]` at the beggining?